examples/stm32f469_discovery/lvgl/main.cpp

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2020, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <modm/board.hpp>
+#include <modm/debug/logger.hpp>
+
+#include <lvgl/lvgl.h>
+
+
+// Set the log level
+#undef	MODM_LOG_LEVEL
+#define	MODM_LOG_LEVEL modm::log::DEBUG
+
+using namespace Board;
+using namespace modm::glcd;
+using namespace Board::ft6;
+using namespace modm::literals;
+
+static modm::Abandonment
+test_assertion_handler(const modm::AssertionInfo &info)
+{
+	MODM_LOG_ERROR << "AssertionInfo.name: " << info.name << "\n";
+	MODM_LOG_ERROR << "AssertionInfo.context: " << info.context << "\n";
+	MODM_LOG_ERROR << "AssertionInfo.behavior: " << info.behavior << "\n";
+	MODM_LOG_ERROR << modm::endl;
+	return modm::Abandonment::DontCare;
+}
+MODM_ASSERTION_HANDLER(test_assertion_handler);
```

Comment:
Isn't this also printed by modm_abandon?

Reply:
I removed the fault handlers from both examples to reduce complexity.